test/flutter_location_listener_test.dart

```diff
@@ -0,0 +1,23 @@
+// import 'package:flutter/services.dart';
+// import 'package:flutter_test/flutter_test.dart';
+// import 'package:flutter_location_listener/flutter_location_listener.dart';
+//
+// void main() {
+//   const MethodChannel channel = MethodChannel('flutter_location_listener');
+//
+//   TestWidgetsFlutterBinding.ensureInitialized();
+//
+//   setUp(() {
+//     channel.setMockMethodCallHandler((MethodCall methodCall) async {
+//       return '42';
+//     });
+//   });
+//
+//   tearDown(() {
+//     channel.setMockMethodCallHandler(null);
+//   });
+//
+//   test('getPlatformVersion', () async {
+//     expect(await FlutterLocationListener.platformVersion, '42');
+//   });
+// }
```

Comment:
can we delete this file?

Reply:
It is useful for when we go to test the plugin